compiler/src/iree/compiler/Codegen/Common/DecomposePackUnPackOps.cpp

```diff
@@ -85,6 +85,12 @@ struct FoldTrailingUnitTranspose
     if (numDropDims == 0)
       return failure();
 
+    // Dropping all dims. Elide now to avoid corner cases.
+    if (numDropDims == inputTy.getRank()) {
```

Comment:
I dont think I follow this one. Can you just provide some context here (just for my information).